packages/rocketchat-livechat/client/views/sideNav/livechat.js

```diff
@@ -35,8 +35,30 @@ Template.livechat.helpers({
 				'name': 1
 			}
 		});
+	},
+	available() {
+		const user = Meteor.user();
+		return {
+			status: user.statusLivechat,
+			icon: user.statusLivechat === 'available' ? 'icon-toggle-on' : 'icon-toggle-off',
+			hint: user.statusLivechat === 'available' ? t('Available') : t('Not_Available')
+		};
+	},
+	livechatAvailable() {
+		const user = Meteor.user();
+
+		if (user) {
+			return user.statusLivechat;
+		}
 	}
 });
 
 Template.livechat.events({
+	'click .livechat-status'() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["results" is defined but never used](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2381151395/issues/source?bid=3199634&fileBranchId=3225821#l58)